Watchman.Discord/Areas/Responses/Controllers/ResponsesController.cs

```diff
@@ -26,8 +26,9 @@ public class ResponsesController : IController
         private readonly MessagesServiceFactory _messagesServiceFactory;
         private readonly UsersRolesService _usersRolesService;
         private readonly Services.ResponsesService _responsesService;
```

Comment:
teraz jeszcze zauważyłem, że jest Services.ResponsesService 
Musi być z tym "Services"? lepiej dać using Services; i odwoływać się po prostu ResponsesService

Reply:
nie to musi pozostać bo jest niejednoznaczność między Watchman.Discord.Areas.Responses.Services.ResponsesService a Devscord.DiscordFramework.Framework.Commands.Responses.ResponsesService

Reply:
a ok